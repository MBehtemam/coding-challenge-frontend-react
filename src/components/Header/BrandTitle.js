import React from 'react';
import styled from 'styled-components';

const BrandTitle = styled.h1`
  color: white;
  font-size: 2rem;
`;
export default BrandTitle;
