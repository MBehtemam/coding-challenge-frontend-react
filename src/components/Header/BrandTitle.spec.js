import React from 'react';
import renderer from 'react-test-renderer';
import BrandTitle from './BrandTitle';

describe('Test suits for BrandTitle Component', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<BrandTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
