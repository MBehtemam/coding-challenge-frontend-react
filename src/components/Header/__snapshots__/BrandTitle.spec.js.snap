// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for BrandTitle Component should match with snapshot 1`] = `
<h1
  className="sc-bdVaJa cIuuyv"
/>
`;
